README.md

```diff
@@ -15,6 +15,30 @@ $ ./selfprofiler.sh > prof.json
 
 In Google Chrome navigate to `about://tracing` and load `prof.json`.
 
+## Installation
+If you would like to install SelfProfiler globally so that you can run it in any directory, you can use `install.sh`.
+
+```sh
+$ ./install.sh
+```
+
+Please restart your terminal or source your environment file after installation is successful.
```

Comment:
🤔 I am not sure how can we make this more convenient. Restarting the terminal is the easiest way, however it is manual.

We can source the corresponding environment file, however everyone has their own preference. I am not sure if we can automatically do this in `install.sh`.

`source ~/.bashrc` or `source ~/.zshrc` or something else

Please let me know if you have a nice solution for this. ☺️